src/dispatch/plugins/dispatch_slack/plugin.py

```diff
@@ -42,6 +45,7 @@
     get_user_email,
     get_user_info_by_id,
     get_user_info_by_email,
```

Comment:
Looks like we don't need this import here anymore. Mind removing it? We can merge the code afterwards.

Reply:
removed